src/app/pages/send-tokens/components/asset-search/asset-search.tsx

```diff
@@ -148,6 +148,7 @@ const AssetSearchField = memo(
             onFocus={() => {
               openMenu();
             }}
+            data-testid={`asset-select`}
```

Comment:
Is there a reason sometimes you add selector constants but other times you don't? It seems like we should always try to add selector constants for new tests?

Reply:
Yes we can always add these that is no problem

Reply:
We should add them before new tests get merged.